### PR TITLE
Update glibc 2.23 2.27

### PR DESCRIPTION
**Pull request by JL2210**
@uberhacker: Thank you for writing this. It needs some work to merge, but it's a definite solution for the issues we're seeing on the stable channel now.

### CODE REVIEW

**Comment by uberhacker** [comment]
I was waiting for the right time to introduce this PR.  Now is almost the time but not quite.  I wanted to wait until the release 72 update has settled in with most devices first.

**Comment by JL2210** [comment]
It has. I got the update arrow over a week ago. It's time.

**Comment by uberhacker** [comment]
It has on YOUR device but I have 2 chromebooks that have not received the update notice yet.  What I don't want is for folks to install this update before it's ready.  Otherwise, we have to tell them to reinstall.

**Comment by JL2210** [comment]
It will be fine. If their Chromebook is still on glibc 2.23 it will install
glibc 2.23. When their device reaches 72 it will upgrade again.

File: packages/glibc.rb [hunk 1]
```diff
@@ -3,29 +3,33 @@
 class Glibc < Package
   description 'The GNU C Library project provides the core libraries for GNU/Linux systems.'
   homepage 'https://www.gnu.org/software/libc/'
-  version '2.23'
-  source_url 'https://ftpmirror.gnu.org/glibc/glibc-2.23.tar.xz'
-  source_sha256 '94efeb00e4603c8546209cefb3e1a50a5315c86fa9b078b6fad758e187ce13e9'
+  version '2.27'
+  source_url 'https://ftpmirror.gnu.org/glibc/glibc-2.27.tar.xz'
```
Comment (JL2210) [inline packages/glibc.rb]: This really should be `https://ftpmirror.gnu.org/libc/glibc-2.27.tar.xz`. `glibc` is a symlink to `libc`.

Comment (uberhacker) [reply packages/glibc.rb]: What exactly should be changed here?

**Comment by JL2210** [comment]
Oh. Can you change that?

**Pull request closed without merging**